gradle/libs.versions.toml

```diff
@@ -1,7 +1,7 @@
 [versions]
 # libraries
-kubernetes-client = "7.1.0"
-devtools-common = "1.9.10"
+kubernetes-client = "7.3.0"
+devtools-common = "1.9.11-SNAPSHOT"
 jackson-core = "2.17.0"
```

Comment:
maybe also bump jackson to 2.19 ? https://central.sonatype.com/artifact/io.fabric8/kubernetes-client-api/7.3.0/dependencies